.factorypath

```diff
@@ -0,0 +1,182 @@
+<factorypath>
```

Comment:
Does this and the classpath changes need to be committed? I imagine some of the classpath changes is fine but maybe we should gitignore this file